src/index.js

```diff
@@ -4,6 +4,8 @@ import './index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
 import 'bootstrap/dist/css/bootstrap.min.css';
+import 'semantic-ui-css/semantic.min.css'
```

Comment:
This gives me compilation error

Reply:
npm install semantic-ui-react
npm install semantic-ui-css